pkg/controller/servicebindingrequest/binding.go

```diff
@@ -339,6 +339,15 @@ func BuildServiceBinder(options *ServiceBinderOptions) (*ServiceBinder, error) {
 	// gather related secret, again only appending it if there's a value.
 	secret := NewSecret(options.DynClient, plan)
 
+	if options.SBR.Spec.ApplicationSelector.BindingPath == nil {
```

Comment:
From what I understood always apply the `PodSpecPath` binding if `BindingPath` is not declared, is it right?

Reply:
Yeah. Correct.

Also, if the user specifies the path to the secret explicitly, I am expecting the user to also specify the containersPath and the volumesPaths , ie, the podSpecPath, explicitly. 

What do you think?